### PR TITLE
Adding a blog post from metal-stack on Gardener ETCD volume migration.

### PR DESCRIPTION
**Pull request by Gerrit91**
Lately, @timebertt asked me to write up a blog post on a topic discussed on the Gardener Slack Channel. It's a case study about the migration of Gardener ETCD volumes in production. I hope you like it.

Once again, I want to state that we really much appreciated the assistance from the Gardener team. :)

### CODE REVIEW

File: website/blog/2020-11/01/_index.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+## How Gardener Manages ETCDs
+
+In our infrastructure, we use the Gardener to provision Kubernetes clusters on bare metal machines in our own data centers using <span style="white-space:nowrap">[metal-stack](https://metal-stack.io/)</span>. Even if the entire stack could be running on-premise, our initial [seed cluster](https://gardener.cloud/documentation/concepts/core-components/api-server/#seeds) and the [metal control plane](https://docs.metal-stack.io/stable/overview/architecture/#Metal-Control-Plane) are hosted on GKE. This way, we do not need to manage a single Kubernetes cluster in our entire landscape manually. As soon as we have Gardener deployed on this initial cluster, we can spin up further Seeds in our own data centers through the concept of [shooted seeds](https://github.com/gardener/gardener/blob/master/docs/usage/shooted_seed.md).
+
+To make this easier to understand, let us give you a simplified picture of how our Gardener production setup looks like:
+
+<img title="Production Setup" src="/blog/2020-11/01/01-001.svg"  style="width:80vw; height:auto">
```
Comment (timebertt) [inline website/blog/2020-11/01/_index.md:42]: Is this correct? In the preview, the image is not displayed correctly...
I assume, standard markup image includes don't work here?

**Review by timebertt — APPROVED** [review]
As you have already included my feedback, no further comments from my side content-wise :)
/lgtm

File: website/blog/2020-11/01/_index.md (hunk 1 above)
Comment (Gerrit91) [reply website/blog/2020-11/01/_index.md:42]: I stole that from other blog articles of this repository. In my preview this was looking good. :) 

Used: 

```
make serve
``` 

Screenshot from 2020-11-18 11-01-55

Comment (timebertt) [reply website/blog/2020-11/01/_index.md:42]: Ok, perfect 👍

**Comment by Gerrit91** [comment]
Mh, just saw that the preview text for the article does not look good. :/

**Comment by Gerrit91** [comment]
I was able to set a better preview text by explicitly defining the `description` field for the article. Should also be fine now. :)

**Review by timebertt — APPROVED** [review]

**Review by g-pavlov — APPROVED** [review]
/lgtm
Please, be aware that the blog will become published and accessible on the website on 2020-11-20 according to the `publishdate` property

**Comment by Gerrit91** [comment]
Hi @g-pavlov, thanks for the review!

Yes, the release date was set intentionally. But of course, you are free to decide when you want to publish such a blog post.

**Comment by timebertt** [comment]
I guess, we can go ahead with the merge then, right @Gerrit91?

**Comment by Gerrit91** [comment]
Yes! :)

**Comment by Gerrit91** [comment]
Thank you. :)

**Pull request merged**